test/check-machines-settings

```diff
@@ -599,8 +599,31 @@ class TestMachinesSettings(VirtualMachinesCase):
 
         b.wait_in_text("#vm-subVmTest1-watchdog-state", "none")
 
+        # General checks for watchdog
+        def openWatchDogDialog():
+            b.click("#vm-subVmTest1-watchdog-button")
+            b.wait_visible("#vm-subVmTest1-watchdog-modal")
+
+        def closeWatchDogDialog(selector):
+            b.click(selector)
+            b.wait_not_present("#vm-subVmTest1-watchdog-modal")
+
+        # Cancel
+        openWatchDogDialog()
+        closeWatchDogDialog("#watchdog-dialog-apply + button")
+
+        # "X" of the dialog
+        openWatchDogDialog()
+        closeWatchDogDialog("#vm-subVmTest1-watchdog-modal button[aria-label=Close]")
```

Comment:
I don't really understand point of these tests. Just opening and closing a modal is not related to watchdog, but more to the modal itself. We use patternfly's modals, so this feels like testing general functionality of pattenfly. Then we also have some custom wrapper at cockpit's [pkg/lib/dialogs.jsx](https://github.com/cockpit-project/cockpit/blob/main/pkg/lib/dialogs.jsx), but watchdog tests still doesn't feel like a place to test it, no?

Reply:
@skobyda we indeed use the dialogs.jsx from lib but we can still forget to add the [onClose property to the Modal](https://github.com/cockpit-project/cockpit-machines/blob/main/src/components/vm/overview/watchdog.jsx#L81), or just wrongly code the Cancel button handlers. These are indeed trivial tests, but they can break on each Modal independently.

Reply:
Fair enough